Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
```

Comment:
There's already a Cargo file in the `parser` directory, we don't need another one here. Delete this file